llama_stack/distribution/access_control/access_control.py

```diff
@@ -105,5 +105,27 @@ def is_action_allowed(
     return False
 
 
+def build_access_denied_message(action: str | None, resource: ProtectedResource | None, user: User | None) -> str:
```

Comment:
```suggestion
def _build_access_denied_message(action: str | None, resource: ProtectedResource | None, user: User | None) -> str:
```

Put it inside `AccessDeniedError`?